core/src/test/java/io/onetable/TestAbstractHudiTable.java

```diff
@@ -334,6 +345,37 @@ public List<HoodieBaseFile> getAllLatestBaseFiles(HoodieTableFileSystemView fsVi
     }
   }
 
+  public void assertMergeOnReadRestoreContainsLogFiles() {
```

Comment:
The method name implies that it will perform some validations. However, the method does not contain any assert statements. This may be intentional, but it is unclear at the moment.

Reply:
@yihua can you provide an update here?

Reply:
@ashvina for MOR table, L357 does the assertion on the Hudi restore metadata to contain at least one log file in the successfully deleted files.  Could you clarify which part does not contain assert statement?